packages/tailwindcss/src/compat/config.test.ts

```diff
@@ -738,6 +738,83 @@ describe('default font family compatibility', () => {
     `)
   })
 
+  test('overriding `fontFamily.sans[1].letterSpacing` sets letter-spacing', async ({ expect }) => {
+    let input = css`
+      @theme default {
+        --default-font-family: var(--font-family-sans);
```

Comment:
Not for this PR but our default theme uses --theme() instead of var(…). We should probably mirror the actual usage in these tests.